platforms/jvm/plugins-java-base/src/main/java/org/gradle/api/plugins/internal/JvmPluginsHelper.java

```diff
@@ -88,8 +88,7 @@ public static void compileAgainstJavaOutputs(AbstractCompile compileTask, final
     }
 
     public static void configureAnnotationProcessorPath(final SourceSet sourceSet, SourceDirectorySet sourceDirectorySet, CompileOptions options, final Project target) {
-        final ConventionMapping conventionMapping = new DslObject(options).getConventionMapping();
-        conventionMapping.map("annotationProcessorPath", sourceSet::getAnnotationProcessorPath);
+        options.getAnnotationProcessorPath().setFrom((Callable<FileCollection>) sourceSet::getAnnotationProcessorPath);
```

Comment:
❓ Should this use `ConfigurableFileCollection.convention()`?

Reply:
Yes, that is true